doc/index.rst

```diff
@@ -16,9 +16,9 @@ Highlights
 **Dipy 0.10** is now available. New features in this release include: 
 
 
-- FVTK 2.0: a new visualization framework. For example, see: 
 - DKI: Thanks to support through the Google Summer of Code, to fund work done by Rafael Henriques, we now have the only open-source implementation of DKI.
```

Comment:
And of course Affine registration is an important new addition to complement the nonlinear registration. Big cool addition.